RELEASING.md

```diff
@@ -11,11 +11,18 @@ In addition to the build pre-requistes:
 
 ## Releasing this project
 
-Firstly ensure that you have completed the [release requirements](https://github.com/kroxylicious/kroxylicious/blob/main/docs/Releasing.adoc#requirements).  Once that is done, use
+Firstly ensure that you have completed the [release requirements](https://github.com/kroxylicious/kroxylicious/blob/main/RELEASING.md#requirements).  Once that is done, use
 the release script to actually perform the release and prepare main for the next development version.
 
 ```shell
 ./scripts/release.sh -k <YOUR_KEY_SHORT_ID> -v <RELEASE_VERSION>
 ```
 
+where `<YOUR_KEY_SHORT_ID>` is the short id of your PGP key
+and `<RELEASE_VERSION>` is a release number such as `0.6.0`.
+
+Once the release is completed, use the [Nexus UI](https://s01.oss.sonatype.org/) to close the staging repository, then release it. That will push the Maven artefacts available
```

Comment:
We should add a note about the timing of merging the version bump. Closing the staging repository involves validation which has the potential to fail and thus need fixing...
Options:

1. Don't merge until the staging repository is closed. 
2. Lazy branching on errors closing the staging repository and re-running the release from there.

Reply:
I'll merge this PR and we can handle this comment in a new one.

Reply:
I opened PR #202